pepper-apis/parent-pom.xml

```diff
@@ -440,6 +440,21 @@
             <artifactId>jetty-servlets</artifactId>
             <version>9.4.28.v20200408</version>
         </dependency>
+        <dependency>
+            <groupId>org.redisson</groupId>
+            <artifactId>redisson</artifactId>
+            <version>3.13.1</version>
+        </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-common</artifactId>
+            <version>4.1.50.Final</version>
+        </dependency>
+        <dependency>
+            <groupId>de.ruedigermoeller</groupId>
+            <artifactId>fst</artifactId>
+            <version>2.57</version>
+        </dependency>
 
```

Comment:
redisson gives us our inplementation of jcache over redis. Netty is a supporting dependency that we had already, but here we specify the version to avoid collisions.
fst is an object serialization library. Used to write the cache objects into redis.